maintainers/scripts/add-test-version.sh

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+info() {
+    echo "INFO:" "$@" > /dev/stderr
+}
+
+warn() {
+    echo "WARNING:" "$@" > /dev/stderr
+}
+
+error () {
+    echo "ERROR:" "$@" > /dev/stderr
+    exit 1
+}
+
+if [ $# == 0 ]; then
+    # git checkout pkgs
+    ALL_PKGS="nix-instantiate --eval -E 'with import ./. {}; lib.attrNames pkgs' --json | jq '.[]' -r"
+    exec nix-shell -p bash ripgrep ag nixpkgs-fmt --run "$ALL_PKGS | xargs -n1 -P4 -- bash $0"
```

Comment:
Why do we need ripgrep and ag?

Reply:
I used those in a previous version of this script to find all packages, but the current version is better. They can be removed.